modules/shared/partials/acid-transactions.adoc

```diff
@@ -43,23 +43,31 @@ If durability is set to `None`, then ACID semantics are not guaranteed.
 // tag::creating[]
 == Creating a Transaction
 
-A core idea of the library is that the application supplies the logic for transaction inside a lambda, including any conditional logic required, and the transactions library takes care of getting the transaction committed.  If the library encounters a transient error, such as a temporary conflict with another transaction, then it can rollback what has been done so far and run the lambda again.
+A core idea of Couchbase transactions is that an application supplies the logic for transaction inside a lambda, including any conditional logic required.
+Couchbase transactions takes care of getting the transaction committed.
+If a transient error occurs, such as a temporary conflict with another transaction, then Couchbase transactions can rollback what has been done so far and run the lambda again.
```

Comment:
"Couchbase transactions can rollback" also reads a bit oddly, to me.  Maybe just "then the transaction will rollback what..."?